### ISSUE TITLE
Conflict with Sodium: UI elements are completely broken underwater

### ISSUE THREAD

**Issue by Felix14-v2** [LaryIsland/ScreenFX#6]
### Description

<details>
 <summary>It is better to see once than read a hundred times:</summary>

![image](https://github.com/LaryIsland/ScreenFX/assets/75726196/546661d0-d385-4477-a4aa-8489d0275b3b)
![image](https://github.com/LaryIsland/ScreenFX/assets/75726196/33846d9a-be89-48f7-9756-5ef637e9e663)
![image](https://github.com/LaryIsland/ScreenFX/assets/75726196/7c66ebc8-06d8-42c5-8864-7d9898e6c682)

</details>

### To reproduce
- Install Sodium (tested with latest, 0.5.8)
- Install ScreenFX
- Make sure you haven't change the underwater overlay opacity setting
- Join a world
- Go underwater
- See the problem

### Workarounds
- Disable underwater overlay transparency (set opacity to 1.0)
- Install [Exordium](https://modrinth.com/mod/exordium) (fixes almost all issues except hotbar)
- Toggle camera view
<details>
 <summary>3rd person view works fine</summary>

![image](https://github.com/LaryIsland/ScreenFX/assets/75726196/fe297a08-2e68-4126-a11e-20e036c6f765)

</details>

### Extra
<details>
 <summary>Seems that particles are also broken:</summary>

https://github.com/LaryIsland/ScreenFX/assets/75726196/8bbb3a2c-2fac-486b-a9e9-93d81eb02983
```
/particle minecraft:explosion_emitter ~ ~2 ~
```
https://github.com/LaryIsland/ScreenFX/assets/75726196/781fb1d7-44cf-4791-b709-92caf35b59cf

</details>

**Comment by LaryIsland** [comment]
I can't seem to reproduce this bug, are you using any other mods?
Are there any sodium options you've changed from default?
What version of minecraft and ScreenFX are you using?

**Comment by Felix14-v2** [comment]
Oh, my apologies, I didn't specify this information. All versions are the latest (from Modrinth): Minecraft 1.20.4, Sodium 0.5.8 (also tested with dev 0.6.0), ScreenFX is 1.4.0. Tested in my test instance, no other mods.

I also did additional testing and found a specific setting in Sodium which caused this – disabled vignette:

https://github.com/LaryIsland/ScreenFX/assets/75726196/459476b3-d6f0-45e6-b852-d31488918a1c

Should I report this to CaffeineMC?


UPD: [No vignette Yes Warning](https://modrinth.com/mod/no-vignette-yes-warning) has the same effect:

https://github.com/LaryIsland/ScreenFX/assets/75726196/7e83f223-5daf-4ff5-acd5-4e77db505fd2
